Add tqdm to Python requirements

sw opened the pull request:
Should fix #289

sw committed:
Add tqdm to Python requirements

Should fix #289

sw commented:
Come to think of it, why is `torchvision torchaudio` there?

bvajk commented:
Did you check wether it really fixes the problem or did you just commit it there because I commented it on the issue?

sw committed:
Remove torchvision torchaudio, add requests

sw commented:
I'm no docker expert and can't figure out how to run the image built by Github (before it's merged, obviously). But I was able to reproduce and fix the problem with a local docker build. I also removed `torchvision torchaudio` as they didn't seem to be needed, and added `requests`.
